packages/dev/s2-docs/src/CodeBlock.tsx

```diff
@@ -78,7 +78,7 @@ export function CodeBlock({render, children, files, expanded, hidden, ...props}:
   }
 
   let content = (
-    <CodePlatter>
+    <CodePlatter {...props}>
```

Comment:
noticed that `type` wasn't being passed through, resulting in the code examples not showing the "open in codesandbox" dropdown menu